test/src/test/groovy/org/openremote/test/notification/NotificationTest.groovy

```diff
@@ -615,21 +615,14 @@ class NotificationTest extends Specification implements ManagerContainerTrait {
         }
         notificationService.notificationHandlerMap.put(emailNotificationHandler.getTypeName(), mockEmailNotificationHandler)
 
-        expect: "the demo users to be created"
-        conditions.eventually {
-            def users = identityService.getIdentityProvider().queryUsers(new UserQuery().realm(new RealmPredicate(keycloakTestSetup.realmBuilding.name)).serviceUsers(false))
-            assert users.size() == 3
-            assert users.count { !TextUtil.isNullOrEmpty(it.email)} == 3
-        }
```

Comment:
Why was this test removed? I assume increasing `3` to `4` would solve the issue here?